AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -418,15 +422,18 @@ - (void)sendStartServiceLog:(NSArray<NSString *> *)servicesNames {
     MSStartServiceLog *serviceLog = [MSStartServiceLog new];
     serviceLog.services = servicesNames;
     [self.logManager processLog:serviceLog forGroupId:kMSGroupId];
+  } else {
+    if (self.startedServicesNamesToLog == nil) {
+      self.startedServicesNamesToLog = [NSMutableArray new];
+    }
+    [self.startedServicesNamesToLog addObjectsFromArray:servicesNames];
```

Comment:
Can we just point servicesNames instead of creating an array?

Reply:
Nope. I've added a test for this case. See https://github.com/Microsoft/AppCenter-SDK-Apple/pull/886/commits/1986a03397e878983645079b4d9f22e8d27b775f